src/CONST.ts

```diff
@@ -2810,6 +2810,17 @@ const CONST = {
      * The count of characters we'll allow the user to type after reaching SEARCH_MAX_LENGTH in an input.
      */
     ADDITIONAL_ALLOWED_CHARACTERS: 20,
+
+    REFERRAL_PROGRAM: {
+        CONTENT_TYPES: {
+            MONEY_REQUEST: 'request',
+            START_CHAT: 'startChat',
+            SEND_MONEY: 'sendMoney',
+            REFERRAL_FRIEND: 'referralFriend',
+        },
+        REVENUE: 250,
+        LINK: 'https://help.expensify.com/articles/new-expensify/getting-started/Referral-Program#gsc.tab=0',
```

Comment:
NAB but this link seems to work and looks better
```suggestion
        LINK: 'https://help.expensify.com/articles/new-expensify/getting-started/Referral-Program',
```